### PR TITLE
Budget App Completion

### PR DESCRIPTION
**Pull request by Obimbo07**
- Deployed the main branch

- Devise authentication done

- added categories and transactions pages

- added flash screen

Added live demo and presentation links to the ReadMe

### CODE REVIEW

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [ ] Your readme is currently missing an important project requirement as shown in the image below, please kindly record this video and add it to your readme before resubmitting :+1: 
pr

File: README.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+- **Flavorful Authentication:** Log in and register with a touch of Devise magic.
+- **Category** Create, edit, and showcase categories 
+- **Display Transactions** a list of transactionsfor each category. 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Start Budgeting!! [Not live yet](#).
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Your live demo is also missing and this is an important requirement for this project please kindly add it before resubmit :+1:

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [ ] You are missing this below requirement for your transactions page, please kindly fix it 
rew

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Obimbo07, 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (Obimbo07) [reply README.md:1]: Done. Thank you for the request.

File: README.md (hunk 1 above)
Comment (Obimbo07) [reply README.md]: I have added the live demo

File: README.md
Comment (Obimbo07) [reply README.md:1]: Image20240226185316

The totals exist on the transactions page on the top upper right.

**Review by Reem-lab — COMMENTED** [review]
Hi @Obimbo07, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't follow the right Gitflow since your PR should be from the last feature branch holding all the files you changed to dev not from dev to main 💪🏻 so kindly create a new branch with all the files changed inside it and create it to dev then send your PR 🔥🔥🔥 you can check this [gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) to have more information about the GitFlow 💪🏻

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**